config/settings/base.py

```diff
@@ -539,7 +539,7 @@
 # By default we're using f"{name} off" format, which works fine with `name` being at least user's first name.
```

Comment:
@Agrendalath Can you update this comment to reflect on the new regex behaviour?

Reply:
Good catch!